frontend/src/pages/team/Brokers/index.vue

```diff
@@ -297,6 +328,25 @@ export default {
         },
         clearUns () {
             this.$store.dispatch('product/clearUns')
+        },
+        getBrokerState () {
+            return brokerAPI.getBrokerStatus(this.team.id, this.activeBrokerId)
+                .then(response => {
+                    if (response?.state?.connected) {
+                        this.brokerState = 'running'
```

Comment:
```suggestion
                        this.brokerState = 'connected'
```

I think using `connected` reinforces the point we are connected to the broker. But this will need adding to `StatusBadge` so it displays the same as `running`.